### DISCUSSION TITLE
dioxus cli 0.6 failed to build desktop on linux

### DISCUSSION THREAD

**Discussion by jessezwd** [DioxusLabs/dioxus#3364] [General]
error: linking with `cc` failed: exit status: 1
note: /usr/bin/ld: cannot find -lxdo: No such file or directory
OS: EndeavourOS
BTW, system dependencies for Tauri installed.

**Comment by Joel-N99** [comment]
Its a linker error. I think its looking for the libxdo library. install the libxdo package and it should work.

**Reply by jessezwd** [reply]
For archlinux/EndeavourOS, install xdotool. It works now, thanks a lot!

**Reply by Joel-N99** [reply]
no problem 👍🏽
